internal/provider/resource_basistheory_proxy_test.go

```diff
@@ -17,7 +17,7 @@ import (
 func TestResourceProxy(t *testing.T) {
 	testAccApplicationName := "terraform_test_application_proxy_test"
 	formattedTestAccReactorCreate := fmt.Sprintf(testAccReactorCreateWithoutApplication, "terraform_test_reactor_proxy_test")
-	formattedTestAccApplicationCreate := fmt.Sprintf(testAccApplicationCreate, testAccApplicationName)
+	formattedTestAccApplicationCreate := fmt.Sprintf(testAccApplicationCreateWithCreateKeyTrue, testAccApplicationName)
```

Comment:
Vault functionality changed after this test was originally written. Vault now requires proxies with Applications to have an Application Key. Otherwise an `ApplicationKeyNotFound` error is raised in Vault causing a 422 terraform error.